tests/OSPSuite.Presentation.Tests/Importer/Presenters/ImporterDataPresenterSpecs.cs

```diff
@@ -272,4 +273,26 @@ public void first_valid_sheet_is_selected()
          A.CallTo(() => _view.SelectTab(_baseSheet)).MustHaveHappened();
       }
    }
+
+   public class When_calculating_format_based_on_invalid_sheet : concern_for_ImporterDataPresenter
+   {
+      protected override void Context()
+      {
+         base.Context();
+         sut.SetDataSource("test_file");
+      }
+
+      protected override void Because()
+      {
```

Comment:
Please use resharper in this file too. There is spaces extra everywhere it seems

Reply:
done